src/excmds.ts

```diff
@@ -4304,6 +4306,9 @@ const KILL_STACK: Element[] = []
           - For example, use `bind ;jg hint -Jc .rc > .r > a` on google.com to generate hints only for clickable search results of a given query
         - -br deprecated, use `-qb` instead
         - -F [callback] - run a custom callback on the selected hint, e.g. `hint -JF e => {tri.excmds.tabopen("-b",e.href); e.remove()}`.
+        - -V create hints for invisible elements. By default, elements outside the viewport when calling :hint are not hinted, this includes them anyways.
```

Comment:
Bug:

1. visit http://txti.es/
2. `:hint -V`
3. Receive `# controller in excmd: TypeError: rect is undefined`. Hints are not numbered

Reply:
It seems allowing "invisible" elements to be hinted breaks some invariants for the Hint class. I can get -V to "somewhat" work by checking if an element has client rects (efd05bf85bbdc4079664d7fc09416bbb1837d409), but the flags are offset for some reason.
![image](https://user-images.githubusercontent.com/13561842/113610772-daa7a480-964d-11eb-9687-7811ac7804fa.png)